framework/wcf/Extensibility/Transport/AzureQueueStorageClient/AzureQueueStorageClient/AzureQueueStorageChannelFactory.cs

```diff
@@ -0,0 +1,117 @@
+﻿//----------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+//----------------------------------------------------------------
+
+using System;
+using System.Collections.ObjectModel;
+using System.ServiceModel;
+using System.ServiceModel.Channels;
+
+namespace Microsoft.Samples.AzureQueueStorage
+{
```

Comment:
I presume this will be released as a nuget package so Samples shouldn't be part of the namespace. In WCF, the namespace naming is usually System.ServiceModel for the top level binding and any types they would reference in their code, so most only need to have a using for that one namespace. Then all the composite pieces used by the binding, and that end users can use to make a custom binding are in System.ServiceModel.Channels. Using this model, you can pick a base namespace which some types are in, and append .Channels for the more inner types such as this. You could use Microsoft.AzureQueueStorage, or Microsoft.ServiceModel.AzureQueueStorage. What aboue Microsoft.ServiceModel.AQS? What namespace is used by the AQS client? Also the nuget package name normally matches this top level name. I think having ServiceModel in there somewhere could be useful for people searching on nuget for just ServiceModel. This is likely a decision to be made by the stakeholders/owners of the final package.

Reply:
This makes sense. Microsoft.Service.model.AQS sounds good to me . its short and meaningful. 
also if you are asking about the namespace of Azure queue. its using Azure.Storage.Queues;